includes/wp-admin/class-advanced-settings-fields.php

```diff
@@ -39,6 +39,17 @@ public static function register_advanced_fields() {
 			array( 'label_for' => 'activitypub_outbox_purge_days' )
 		);
 
+		if ( ! defined( 'ACTIVITYPUB_SEND_VARY_HEADER' ) ) {
```

Comment:
I think this is the third way of how we deal with settings that also have a constant, after displaying a note instead of the setting and disabling the setting. Should we settle on one?

I guess in this instance we introduce the setting and there's no expectation for it to be there, unlike the others.